db/seeds.rb

```diff
@@ -187,7 +187,6 @@
         oidc_url: "https://blackboard.com/",
       },
     ],
-
```

Comment:
Revert this change so it doesn't cause downstream merging headaches?

Reply:
done